src/packages.builds

```diff
@@ -4,13 +4,38 @@
 
   <PropertyGroup>
     <PackageReportDir Condition="'$(PackageReportDir)' == ''">$(BinDir)pkg/reports/</PackageReportDir>
+    
+    <!-- 'StabilizePackageVersion' is used in conjunction with 'PackageVersionStamp' to determine whether or not a label and build number should be appended to a package name. -->
+    <!-- Default behavior when not building stable packages is to build both WCF and SvcUtil packages. -->
+    <!-- 'StabilizePackageVersion' is a flag set manually when stable packages are needed. -->
+    <!-- 'BuildWcfPackages' and 'BuildSvcUtilPackage' should also be explicitly set when stable packages are desired, otherwise no packages will be built. -->
+    <BuildWcfPackages Condition="'$(StabilizePackageVersion)' != 'true' AND '$(BuildWcfPackages)' == ''">true</BuildWcfPackages>
```

Comment:
It's probably clearer to call this `BuildServiceModelPackages" since conceptually, everything in WCF repo are WCF packages.

Reply:
True